jwt/validate.go

```diff
@@ -195,6 +197,14 @@ func ErrTokenNotYetValid() ValidationError {
 	return errTokenNotYetValid
 }
 
+func ErrTokenInvalidAudience() ValidationError {
```

Comment:
In hindsight, `errTokenNotYetValid` was a misnomer which should have been corrected in 2.0.0...
This should also be `ErrInvalidAudience()`, as well as the error value itself.